packages/angular-cli/blueprints/ng2/files/angular-cli.json

```diff
@@ -54,6 +54,11 @@
       "module": false,
       "pipe": true,
       "service": true
+    },
+    "serve": {
```

Comment:
Since you already have good default values, those are not needed and just add noise to this file. I'd like the default config to grow slimmer.